expenses-manager-frontend/src/main/frontend/src/app/services/category-service/category.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+import { Observable } from 'rxjs';
+
+export interface Category {
+  id: number;
+  description: string;
+  color: string;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CategoryService {
+
+  private apiUrl= environment.baseUrl;
+  constructor(private http: HttpClient) { }
+
+  public getAllCategories(): Observable<string[]>{
+    return this.http.get<string[]>(`${this.apiUrl}categories`);
```

Comment:
Unde am definit baseUrl-ul sterge / de la finalul lui pentru ca aici arata ciudat si e confuz ca this.apiUrl e concatenat cu categories de exemplu